### PR TITLE
detect-bytetest/v1: add tests

### PR DESCRIPTION
**Pull request by TheKharleeci**
detect-bytetest-02: DetectByteTestTestPacket01-05
detect-bytetest-03: DetectByteTestTestPacket06

### CODE REVIEW

**Comment by TheKharleeci** [comment]
Corresponding Suricata PR: https://github.com/OISF/suricata/pull/6945

File: tests/detect-bytetest-03/test.rules [hunk 1]
```diff
@@ -0,0 +1,3 @@
+alert dns any any -> any any (msg:"Byte test against first byte"; byte_test:1,=,0x07,0,bitmask 0xF8; sid:1;)
```
Comment (inashivb) [inline tests/detect-bytetest-03/test.rules:1]: `bitmask` was not implemented in `5.0.x`, please add a min-version check in this test for 6.

**Review by inashivb — CHANGES_REQUESTED** [review]
One small requested inline change wrt version requirements.

**Comment by TheKharleeci** [comment]
Followed By: #720

**Pull request closed without merging**